ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -1991,9 +1991,9 @@ private Boolean SetRange(Object rangeObject)
             {
                 if (!(asRange is XLRow || asRange is XLColumn))
                 {
-                    var maxRows = asRange.RowCount();
-                    var maxColumns = asRange.ColumnCount();
-                    Worksheet.Range(_rowNumber, _columnNumber, maxRows, maxColumns).Clear();
+                    var maxRow = _rowNumber + asRange.RowCount() - 1;
+                    var maxColumn = _columnNumber + asRange.ColumnCount() - 1;
+                    Worksheet.Range(_rowNumber, _columnNumber, maxRow, maxColumn).Clear();
```

Comment:
This was fixed in https://github.com/ClosedXML/ClosedXML/pull/1022/commits/2cc52eba90992671e4d396f3b12a7207b9f7f5fe , right?

Reply:
I noticed that too. I am going to rebase anyway, so this will be replaced with your solution.